src/pages/BusinessConditionsPage.js

```diff
@@ -4,7 +4,122 @@ export default class BusinessConditionsPage extends React.Component {
   render() {
     return (
       <div>
-        Obchodní podmínky
```

Comment:
těžko říct co s tímhle a naší potenciální otevřeností pro další jazykové mutace, je to takhle finální, nebo to půjde z backendu? nějaký nápad jak to otevřít pro lokalizace (přidávat to do toho lokalizačního souboru by byl trochu masakr)?